### PR TITLE
Feature/history page

### PR DESCRIPTION
**Pull request by jtranssf**

### CODE REVIEW

File: src/components/search-list/search-list.html [hunk 1]
```diff
@@ -1,47 +1,58 @@
...
+    <p (click)="emptyHistory=!emptyHistory"><u><i>View Mock Data</i></u></p> 
+  </ion-label>
   
-      </ion-item>
-    </ion-list>
-  </div>
+ <!---- Display our test history if it is not empty ---->
+  <ion-list *ngIf="!emptyHistory">
+    <!---- Each Test Card (test info and mini chart) ---->
+    <ion-item class="item display-block" *ngFor="let obj of ourList | testsByTime : searchTime " (click)="selectedItem($event, obj)">
```
Comment (phortonssf) [inline src/components/search-list/search-list.html]: We can chain pipes here so ourList can be filtered by the search value from the search bar and the date as well.
for example
 *ngFor="let obj of ourList | searchTermPipe: searchTerm | testsByTime : searchTime

File: src/components/search-list/search-list.ts [hunk 2]
```diff
@@ -84,8 +94,14 @@ export class SearchListComponent {
     this.navCtrl.push(ResultsPage);
   }
   
+  //Go to the TestsLists Page if user clicks on the option
   goToTestsPage(){
     this.navCtrl.push(TestListsPage);
   }
+  
+  //When user selects a new time range, update their tests view
+  changeTimeView(){
```
Comment (phortonssf) [inline src/components/search-list/search-list.ts]: This function here and where it is called on the html, I do not believe are needed.

Comment (jtranssf) [reply src/components/search-list/search-list.ts]: K, updates made

**Review by phortonssf — APPROVED** [review]
Gj on the pipes!

**Pull request merged**